python/py-klein/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
```

Comment:
it looks like you're not actually using the select PortGroup or...?

Reply:
If that's the case, perhaps this should be fixed before a merge.

Reply:
oh yeah, maybe I took a Portfile example where *_select subport also exists. I'll fix this for all the python packages that were submitted as PR.

Also what is the best way to push changes ? force push to branch as single commit using commit --amend ?

Reply:
Force push, yah.